packages/typescript/internal/ts_project.bzl

```diff
@@ -641,26 +640,19 @@ def ts_project_macro(
                 # Users get this dependency transitively from @bazel/typescript
                 # but that's our own code, so we don't.
                 "@npm//protobufjs",
+                "@npm//typescript",
                 # END-INTERNAL
                 Label("//packages/typescript/internal/worker:worker"),
-                Label(worker_tsc_bin),
-                Label(worker_typescript_module),
                 tsconfig,
             ],
             entry_point = Label("//packages/typescript/internal/worker:worker_adapter"),
             templated_args = [
-                "$(execpath {})".format(Label(worker_tsc_bin)),
                 "--project",
                 "$(execpath {})".format(tsconfig),
```

Comment:
According to the https://docs.bazel.build/versions/3.7.0/persistent-workers.html#implementing-persistent-workers, such configurations should be passed as startup options (arguments with @ prefix). Could you please share why you decide to pass the project argument for every worker invocation? We want to reuse the same worker instance across different packages, or such an approach has another purpose?

Reply:
I pass it only for the standalone varaint of the worker process. I don't think its necessary, but something is wrong with my current logic if I omit it

Reply:
Fixed it!